integration/sandbox_cgroup/sandbox_cgroup_test.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This will enable the sandbox_cgroup_only
+# to true in order to test that docker is
+# working properly when this feature is
+# enabled
+
+set -e
+
+dir_path=$(dirname "$0")
+source "${dir_path}/../../lib/common.bash"
+source "${dir_path}/../../.ci/lib.sh"
+tests_repo="${tests_repo:-github.com/kata-containers/tests}"
+
+function setup() {
+        clean_env
+        check_processes
+        extract_kata_env
+        # Enable sandbox_cgroup
+        sudo sed -i 's/sandbox_cgroup_only=false/sandbox_cgroup_only=true/g' "${RUNTIME_CONFIG_PATH}"
```

Comment:
You may want to use https://github.com/kata-containers/tests/blob/master/.ci/run.sh#L28